logitrail-woocommerce.php

```diff
@@ -1022,19 +1038,33 @@ function update_product() {
         }
     }
 
-    public static function  logitrail_shipping_enabled($product_id) {
-        $product = wc_get_product( $product_id );
-        // Disable shipping on some product types
-        if ( in_array($product->get_type(), array("bundle")) ) {
+    /**
+     * @param WC_Product $product
+     * @param bool $ignore_bundled
+     * @return bool
+     */
+    public function logitrail_is_virtual($product, $ignore_bundled = false) {
+        // We don't want to ship virtual products, but bundled products are virtual so exclude them
```

Comment:
"We don't want to ship virtual products, so we check is the product really virtual, or just part of bundle and marked virtual for this"